tests/haskell/GhcModBearTest.py

```diff
@@ -0,0 +1,56 @@
+from bears.haskell.GhcModBear import GhcModBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+# A simple hello world program!
+good_file1 = """
+main :: IO()
+main = putStrLn "Hello World!"
+"""
+
+# Here putStrLn takes only String type
+bad_file1 = """
+main :: IO()
+main = putStrLn 3
+"""
+
+# Takes a file and prints the number of lines
+good_file2 = """
+main :: IO()
+main = interact lineC
+    where lineC input = show (length(lines input))
+"""
+
+# Same program but paranthesis is missing
+bad_file2 = """
```

Comment:
can we still give them proper names? Enumerating isn't helpful. Good names are better than comments because they appear wherever the variable is used